.github/workflows/run-tests.yml

```diff
@@ -59,7 +60,10 @@ jobs:
       - run: python -V 2>&1 | tee test_linux_artifacts_python_${{ matrix.python-version }}/python_version.txt
       - run: pip install -e .[develop] 2>&1 | tee test_linux_artifacts_python_${{ matrix.python-version }}/install.txt
       - run: flake8
-      - run: pytest -n 2 -m "not installation" 2>&1 | tee test_linux_artifacts_python_${{ matrix.python-version }}/test_report.txt
+      - run: |
+          sudo update-locale LC_ALL=C
```

Comment:
What is this `update-locale` thing? You shouldn't need to try and change the default locale for the whole machine. Just do an `export LC_ALL=C`?

Reply:
Meh. Gonna add encoding to all open file calls anyway

Reply:
The thing we want to test is that it works fine with a default character set other than utf-8, as that is what we keep running into (see  linked issues).